cocotb/share/makefiles/simulators/Makefile.icarus

```diff
@@ -67,7 +67,7 @@ COMPILE_ARGS += -f $(SIM_BUILD)/cmds.f -g2012 # Default to latest SystemVerilog
 # Compilation phase
 $(SIM_BUILD)/sim.vvp: $(VERILOG_SOURCES) $(CUSTOM_COMPILE_DEPS) | $(SIM_BUILD)
 	@echo "+timescale+$(COCOTB_HDL_TIMEUNIT)/$(COCOTB_HDL_TIMEPRECISION)" > $(SIM_BUILD)/cmds.f
-	$(CMD) -o $(SIM_BUILD)/sim.vvp -D COCOTB_SIM=1 $(TOPMODULE_ARG) $(COMPILE_ARGS) $(EXTRA_ARGS) $(VERILOG_SOURCES)
+	$(CMD_PREFIX) $(CMD) -o $(SIM_BUILD)/sim.vvp -D COCOTB_SIM=1 $(TOPMODULE_ARG) $(COMPILE_ARGS) $(EXTRA_ARGS) $(VERILOG_SOURCES)
```

Comment:
Icarus also has a debug target which should get the `$(CMD_PREFIX)`.

Reply:
Yeah. And this `CMD_PREFIX` was added to the compilation phase too.

This makefile is a bit weird in the sense that `CMD_PREFIX` now _doesn't_ prefix `$(CMD)`. So perhaps `CMD_PREFIX` is still a misnomer.